rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -531,7 +535,7 @@ private Integer createPendingFederation(boolean dryRun) {
             return FederationChangeResponseCode.SUCCESSFUL.getCode();
         }
 
-        currentPendingFederation = new PendingFederation(Collections.emptyList());
+        PendingFederation currentPendingFederation = new PendingFederation(Collections.emptyList());
```

Comment:
currentPendingFederation? 🤔 I thought we checked above that there is not an existing pending federation

Reply:
wdym? the check [here](https://github.com/rsksmart/rskj/pull/2629/files#diff-4a4b5a028a8ff057d136592eae64d90f7d7bf630169a6b6e3f778505756f83e8R513) is that the pending federation already exists 🤔